### DISCUSSION TITLE
Is it possible to translate route segments?

### DISCUSSION THREAD

**Discussion by branislavbrincko** [amannn/next-intl#318] [Q&A]
Is is possible to translate urls using next-intl, like: 
- /en/manual 
- /cs/navody
I haven't found anything regarding this usecase.

**Comment by amannn** [comment]
Yes, that's possible. Please see [the docs for localizing pathnames.](https://next-intl-docs.vercel.app/docs/next-13/middleware#localizing-pathnames)
